manager/manager.go

```diff
@@ -806,36 +806,31 @@ func (m *Manager) watchForClusterChanges(ctx context.Context) error {
 	return nil
 }
 
-// rotateRootCAKEK will attempt to rotate the key-encryption-key for root CA key-material in raft.
-// If there is no passphrase set in ENV, it returns.
-// If there is plain-text root key-material, and a passphrase set, it encrypts it.
-// If there is encrypted root key-material and it is using the current passphrase, it returns.
-// If there is encrypted root key-material, and it is using the previous passphrase, it
-// re-encrypts it with the current passphrase.
-func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
-	// If we don't have a KEK, we won't ever be rotating anything
+// decryptRootCAKEK will attempt to decrypt the key-encryption-key for root CA key-material in raft.
+// We previously supported (1) encrypting the root CA key in raft using a passphrase provided by an environment variable,
+// and (2) hitless passphrase rotation if the user wanted to provide us a previous passphrase as well as a current
+// passphrase (we'd rotate the encryption).  We now no longer support encrypting the key in raft, since we will
+// rely on TLS in transit and raft log encryption at rest, but this will read the passphrase from both
+// of the environment variables in order to decrypt, and store decrypted, the root CA key.
+func (m *Manager) decryptRootCAKEK(ctx context.Context, clusterID string) error {
+	// If we don't have a KEK, we won't need to, or can't, decrypt anything
 	strPassphrase := os.Getenv(ca.PassphraseENVVar)
 	strPassphrasePrev := os.Getenv(ca.PassphraseENVVarPrev)
 	if strPassphrase == "" && strPassphrasePrev == "" {
```

Comment:
Should we just ignore this and try to resolve the KEK?

Reply:
Er, sorry, could you explain?  Do you mean ignore `ca.PassphraseENVVarPrev`, and just expect someone to give us the current passphrase?